docs/search/criteria_reference/shipment_updatedat_criterion.md

```diff
@@ -0,0 +1,24 @@
+---
+description: Shipment UpdatedAt Criterion
+edition: commerce
+---
+
+# Shipment UpdatedAt Criterion
+
+The `UpdatedAtCriterion` Search Criterion searches for shipments based on the date when their status was updated.
+
+## Arguments
+
+- `updatedAt` - date to be matched, provided as a `DateTimeInterface` object
+- `operator` - optional operator constant (EQ, GT, GTE, LT, LTE)
```

Comment:
AFAIU these are actually not constants, but strings, as in the example. The same in payments. Ref. https://github.com/ibexa/checkout/blob/main/src/contracts/Shipment/Query/Criterion/CreatedAt.php#L17